bench/src/strategies/tc1.rs

```diff
@@ -0,0 +1,88 @@
+use anyhow::Context;
+use solana_rpc_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::transaction::Transaction;
+use solana_sdk::{commitment_config::CommitmentConfig, signature::Keypair};
+
+use crate::helpers::{BenchHelper, Rng8};
+
+use super::Strategy;
+use crate::cli::{LiteRpcArgs, RpcArgs};
+
+#[derive(Debug, serde::Serialize)]
+pub struct Tc1Result {
+    rpc_slot: u64,
+    lite_rpc_slot: u64,
+}
+
+/// send 2 txs (one via LiteRPC, one via Solana RPC) and compare confirmation slot (=slot distance)
+#[derive(clap::Args, Debug)]
+pub struct Tc1 {
+    #[command(flatten)]
+    rpc_args: RpcArgs,
+
+    #[command(flatten)]
+    lite_rpc_args: LiteRpcArgs,
+}
+
+impl Tc1 {
+    async fn create_tx(
+        &self,
+        rpc: &RpcClient,
+        payer: &Keypair,
+        rng: &mut Rng8,
+    ) -> anyhow::Result<Transaction> {
+        let hash = rpc.get_latest_blockhash().await?;
```

Comment:
Do a seperate task to get blockhash and use it everywhere. This added additional time.

Reply:
FIXED